packages/wallet/api/src/lib-wallet.js

```diff
@@ -1528,6 +1528,23 @@ export function makeWallet({
     return offerResult.uiNotifier;
   }
 
+  /**
+   * Gets the notifiers from an offer's result.
+   *
+   * @param {string} rawId - The offer's raw id.
+   * @param {string} dappOrigin - The origin of the dapp the offer came from.
+   */
+  async function getNotifiers(rawId, dappOrigin = 'unknown') {
```

Comment:
Can you please name this `getPublicNotifiers`, and have it expose `offerResult.publicNotifiers`?  The reason is that we want to prevent the case where the contract doesn't intend to expose the notifiers to the (untrusted) dapp UI.

`getUINotifiers` and `offerResult.uiNotifiers` would also be an option, if we want to be consistent with the original decision for disambiguation.

Reply:
Changed to `getPublicNotifiers`/`offerResult.publicNotifiers`.

@turadg pointed out that `uiNotifiers` might not necessarily just be used by a UI, so I think `public` works best.